thunder/tests/opinfos.py

```diff
@@ -2232,6 +2232,14 @@ def fmod_sample_input_generator(op, device, dtype, requires_grad, **kwargs):
 )
 elementwise_binary_ops.append(logical_and_opinfo)
 
+logical_or_opinfo = OpInfo(
+    clang.logical_or,
+    dtypes=(datatypes.exact, datatypes.floating),
```

Comment:
Sorry, I missed this on the first pass.  Same question as https://github.com/Lightning-AI/lightning-thunder/pull/1859#discussion_r1988939031.